focus-android/app/src/main/res/values-sw480dp/dimens.xml

```diff
@@ -2,7 +2,8 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<resources xmlns:tools="http://schemas.android.com/tools">
-    <dimen name="tablet_bottom_sheet_width" tools:ignore="MissingDefaultResource">480dp</dimen>
-    <dimen name="tablet_bottom_sheet_peekheight" tools:ignore="MissingDefaultResource">320dp</dimen>
+<resources>
+    <dimen name="firstrun_container_width">256dp</dimen>
+    <dimen name="tablet_bottom_sheet_width">384dp</dimen>
+    <dimen name="tablet_bottom_sheet_peekheight">256dp</dimen>
```

Comment:
Should these values be changed? If these are for a smallest width of 480dp, wouldn't we want the `tablet_bottom_sheet_width` to match that width, for example?

Reply:
This value is calculated proportionally to the sw600dp values. (commit message also states that)